contracts/round/RoundFactory.sol

```diff
@@ -29,6 +29,14 @@ contract RoundFactory is OwnableUpgradeable {
   /// @notice Address of the RoundImplementation contract
   address public roundImplementation;
 
+  /// @notice Unused storage slot to allow for upgrade to new variables
+  /// @custom:oz-renamed-from protocolTreasury
+  address payable public _unusedSlot1;
```

Comment:
I am tempted to just update this to `alloSettings` but would be forced to make this payable which doesn't make sense. A settings contract shouldn't be payable